Exiled.Installer/Exiled.Installer.csproj

```diff
@@ -16,7 +16,7 @@
     <OutputPath>bin\$(Configuration)\</OutputPath>
     
     <AssemblyTitle>Exiled online installer</AssemblyTitle>
-    <Version>3.2.0</Version>
+    <Version>3.3.0</Version>
```

Comment:
Why did you bump minor version for a fix?

Reply:
because I forgot to bump the version when I did my initial changes before releasing 2.2.0, Idk why I bumped minor instead of patch though, I will fix that.